meetings/2024-02/README.md

```diff
@@ -0,0 +1,35 @@
+# 100th meeting of Ecma TC39 Summary
+
+## Proposals
+
+| Advanced to | Proposal                 |
+| :---------: | ------------------------ |
+|   Stage 4   | ArrayBuffer transfer     |
+|   Stage 3   | Base64 conversion        |
+|  Stage 2.7  | ShadowRealm              |
```

Comment:
```suggestion
|   Stage 2.7 | ShadowRealm              |
```
?

Reply:
`:---------:` column header === centered.  are you suggesting to right align the col?

Reply:
i'm suggesting to left-align it; it reads more nicely than centered imo, and matches the right column

Reply:
![Screenshot 2024-03-26 at 1 57 12 PM](https://github.com/tc39/notes/assets/15326526/f6e0f89b-3d13-4574-8639-14a9f637e4b8)

![Screenshot 2024-03-26 at 1 57 06 PM](https://github.com/tc39/notes/assets/15326526/aa41b728-3b05-4b96-85a4-f8a55d5dc8f9)

Reply:
your suggested change right aligns it though.. 😕 

default alignment is fine. I prefer centered but I don't care that much. you ask, you get

Reply:
hm, my suggestion should have moved the "stage 2.7" to the right, so that the S aligns with the other S's in the column.

thanks!

Reply:
in the raw markdown, sure, but everything else would still be centered in raw markdown, and the rendered version would still be centered due to the header

Reply:
ahh, ok, missed that facet.